src/panel/PanelManager.jsx

```diff
@@ -149,9 +149,12 @@ export default class PanelManager extends React.Component {
         || parseQueryString(document.location.search)['pt'] === 'true';
 
       router.addRoute('Routes', '/routes(?:/?)(.*)', (routeParams, options) => {
+        const params = parseQueryString(routeParams);
+        params.details = params.details === 'true';
+        params.activeRouteId = Number(params.selected);
```

Comment:
I think we should handle the case where this returns a `NaN`. Can we default to 0 ?

Reply:
Done!